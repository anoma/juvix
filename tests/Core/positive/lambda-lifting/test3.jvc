def t2 := \r \s r (\x \y s y (\z z y x (\w w x) (\e y e x y)));
