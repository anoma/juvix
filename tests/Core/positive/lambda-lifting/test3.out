-- IdentContext
def const := λx λy x$1
def id := λx x$0
def t3 := λr λs const !5 (id (!6 r$1))
def 5 := λx x$0
def 6 := λr λx r$1
